packages/yoga/src/Heading/native/Heading.jsx

```diff
@@ -0,0 +1,134 @@
+import { arrayOf, bool, func, shape, string } from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+import { ArrowLeft } from '@gympass/yoga-icons';
+import Box from '../../Box';
+import Text from '../../Text';
+import Button from '../../Button';
+import Icon from '../../Icon';
+import TitleContainer from './TitleContainer';
+
+const HeadingContainer = styled(Box)`
+  ${({
+    theme: {
+      yoga: {
+        components: { heading },
+      },
+    },
+  }) => `
+    padding: ${heading.paddingTop}px ${heading.paddingRight}px ${heading.paddingBottom}px ${heading.paddingLeft}px;
+`}
+`;
+
+const SubContainer = styled(Box)``;
+const RightIconContainer = styled(Box)``;
+
+const RightIcon = styled(Button.Link)`
+  ${({
+    isFirst,
+    theme: {
+      yoga: {
+        components: { heading },
+      },
+    },
+  }) => `
+  padding-left: ${isFirst ? heading.rightIcons.paddingLeft : 0}px;
+`}
+`;
+const PageTitle = styled(Text.Small)``;
+
+const Heading = props => {
+  const {
+    backButton,
+    pageTitle,
+    rightIcons,
+    title,
+    subtitle,
+    avatar,
+    smallAvatar,
+  } = props;
+
+  return (
+    <HeadingContainer w="100%">
+      <SubContainer
+        flexDirection="row"
+        alignItems="center"
+        justifyContent="space-between"
+      >
+        {backButton && (
+          <Button.Link onPress={backButton.onPress}>
+            <Icon
+              as={ArrowLeft}
+              width="medium"
+              height="medium"
+              fill="primary"
+            />
+          </Button.Link>
+        )}
+        <PageTitle>{pageTitle}</PageTitle>
+        <RightIconContainer flexDirection="row">
+          {rightIcons &&
+            rightIcons.map((rightIcon, index) => (
+              <RightIcon
+                onPress={rightIcon.onPress}
+                isFirst={index === 0}
```

Comment:
I guess you don't need this prop here, it'd be more elegant if you could solve it with the `:first` pseudo-class 
is it possible?